code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -48,7 +48,7 @@
 	if (malf_ai.len < required_enemies)
 		return 0
 	for(var/datum/mind/ai_mind in malf_ai)
-		ai_mind.assigned_role = "MODE"
+		ai_mind.assigned_role = "malfunctioning AI"

```

Comment:
It needs to match the special_role, which was also lowercased. I took the path of least resistance.